packages/lesswrong/server/migrations/20240526T111308.set_not_null_clientId_2.ts

```diff
@@ -0,0 +1,78 @@
+import { ClientIds } from "@/lib/collections/clientIds/collection";
+import { updateIndexes } from "./meta/utils";
+import { idxName } from "../manualMigrations/2024-05-26-setNotNullClientIds";
```

Comment:
I don't love importing from manual migrations here. It violates an expectation that those files are optional for mature dbs and can be deleted if desired.

Reply:
👍 Good point I'll just define it in both places